### DISCUSSION TITLE
Project code

### DISCUSSION THREAD

**Discussion by DoctorChinnery** [Part-II-Computational-Physics/.github#92] [General]
Good afternoon,
For my project there are several blocks of simulation code that take way too long to be run within the report. I plan on running them separately and saving the output data as a .npy in the repository. Should this code that is being run still be included in the body of the report, or included as a separate .py that is clearly marked and referred to?
Thanks

**Comment by AustenLamacraft** [comment]
Separate `.py` files for simulation and data as `.npy`. Then you can just refer to the simulation code in the report
